### DISCUSSION TITLE
Melhores tecnologias para montar um Ecommerce

### DISCUSSION THREAD

**Discussion by pedroribeirodev** [frontendbr/forum#1531] [Discussão]
Tudo bem galera?
Estou reformulando um ecommerce e quero aplicar novas tecnologias nele, atualmente ele tá com WooCommerce, quero utilizar React e NodeJS nele. Quero saber de quem tem mais experiência as bibliotecas mais indicadas para esse projeto, se vocês tem conselho para isso.

Thanks!

**Comment by faelribeiro22** [comment]
Cara começa olhando o Next.js. Por ser um server side rendering ele auxilia muito na melhoria de SEO do seu ecommerce usando o react.

**Comment by glauberm** [comment]
Eu te recomendaria usar a [API REST do WooCommerce](https://woocommerce.github.io/woocommerce-rest-api-docs) alimentando o [Gatsby](https://www.gatsbyjs.org/) através do plugin [gatsby-source-wordpress](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress).

**Comment by pedroribeirodev** [comment]
> Eu te recomendaria usar a [API REST do WooCommerce](https://woocommerce.github.io/woocommerce-rest-api-docs) alimentando o [Gatsby](https://www.gatsbyjs.org/) através do plugin [gatsby-source-wordpress](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-wordpress).

Aí eu teria que usar o WordPress né? To querendo sair dele hahaha

**Comment by glauberm** [comment]
@pedroribeirodev Ah, sim. Então no caso você não quer bem reformular seu ecommerce, né, você quer fazer outro, hahahahaha. Você quer fazer do zero? Se não, dá uma olhada no [Reaction](https://github.com/reactioncommerce/reaction).

**Comment by pedroribeirodev** [comment]
> @pedroribeirodev Ah, sim. Então no caso você não quer bem reformular seu ecommerce, né, você quer fazer outro, hahahahaha. Você quer fazer do zero? Se não, dá uma olhada no [Reaction](https://github.com/reactioncommerce/reaction).

Irei dar uma olhada, mas a ideia que você deu anteriormente, eu vou dar uma olhada tb. Eu consigo aproveitar a atual base, para utilizar a API do woocommerce?

**Comment by glauberm** [comment]
@pedroribeirodev O que eu te recomendaria é a minha primeira sugestão mesmo. Acho que você vai ver que usando a REST API o Wordpress é bem mais prazeroso de se trabalhar. O WooCommerce é integrado com ela nativamente desde a versão 2.6, então sim, você consegue aproveitar toda a sua base atual.

**Comment by felipefialho** [comment]
Iria de [Next.js](https://nextjs.org/)
